libraries/AP_HAL_ChibiOS/hwdef/MazzyStarDrone/hwdef.dat

```diff
@@ -0,0 +1,179 @@
+# for Mazzy Star Drone FC.
+
+# MCU class and specific type
+MCU STM32F7xx STM32F745xx
+
+# board ID for firmware load
+APJ_BOARD_ID 188
+
+# crystal frequency, setup to use external oscillator
+OSCILLATOR_HZ 8000000
+
+define STM32_LSECLK   32768U
+define STM32_LSEDRV   (3U << 3U)
+
+define STM32_PLLSRC     STM32_PLLSRC_HSE
+define STM32_PLLM_VALUE 8
+define STM32_PLLN_VALUE 432
+define STM32_PLLP_VALUE 2
+define STM32_PLLQ_VALUE 9
+
+FLASH_SIZE_KB 1024
+
+# leave 2 sectors free
+FLASH_RESERVE_START_KB 96
+
+# board voltage
+STM32_VDD 330U
+
+# only one I2C bus
+I2C_ORDER I2C1
+
+# order of UARTs (and USB), USART3 should be in second place to map order with the board's silk screen
+SERIAL_ORDER OTG1 UART4 USART1 USART2 USART3 UART7 USART6
```

Comment:
UART4 is listed as GPS2 below, but is setup as SERIAL1 here

Reply:
I have updated the SERIAL_ORDER, USART3 and USART4 is reverse in the board